src/_utils.js

```diff
@@ -524,7 +524,8 @@ export const processCss = (stylesInfo, options) => {
     vendorPrefixes,
     sourceMaps: useSourceMaps,
     isGlobal,
-    filename: fileInfo.filename
+    filename: fileInfo.filename,
```

Comment:
Mh can't remember what is the difference between `filename` and `sourceFileName` 🤔 Do you mind checking in the babel docs or trying to console.log both in a plugin? Maybe `filename` is enough?